packages/core/getUserAvailability.ts

```diff
@@ -256,8 +256,8 @@ export async function getUserAvailability(
       const startTime = dayjs.utc(override.startTime);
       const endTime = dayjs.utc(override.endTime);
       return {
-        start: dayjs.utc(override.date).hour(startTime.hour()).minute(startTime.minute()).toDate(),
-        end: dayjs.utc(override.date).hour(endTime.hour()).minute(endTime.minute()).toDate(),
+        start: dayjs.tz(override.date, timeZone).hour(startTime.hour()).minute(startTime.minute()),
```

Comment:
My date overrides are shown on a wrong day. I have organizer and booker in the same timezone (Eastern Time). The date override is always applied one day before the actual day. 

It looks like setting the hour and minute changes the date to the day before